### ISSUE TITLE
Parsing rgb() causes an exception

### ISSUE THREAD

**Issue by lukas-we** [scssphp/scssphp#94]
When using the `rgb()` function in an SCSS file, an exception "Missing argument alpha" is thrown. The compilation still succeeds, but I think this is quite irritating.

The exception is raised because `sortNativeFunctionArgs()` iterates through all prototypes of libRgb, which includes the rgba-notation. I wonder, if this is actually necessary, as the `rgb()`-function is unambiguous in its arguments and requires exactly three of them (see https://www.w3schools.com/cssref/func_rgb.asp).

**Comment by stof** [comment]
That's not true of the Sass rgb function. You are linking to some CSS reference, but this is not what scssphp is about. See https://sass-lang.com/documentation/modules#rgb

Also, where do you see this exception ?

**Comment by lukas-we** [comment]
Ah sorry, I should have checked the Sass documentation.

I have investigated a bit further and the fault does not seem to be directly with scssphp.

The problem I've got is, that my Php process just seems to die at some point. It reaches `sortNativeFunctionArgs` there `applyArguments` is called, where an exception is raised, because of the missing alpha value. This exception is caught in `sortNativeFunctionArgs`. So far, so good.

But after processing some rgb functions in the Scss files (ie. it works fine at first), the process seems to die, when the exception is raised. I checked which parts of the code are reached by printing some output into a file at specific points in the code and the line in `throwError` just before actually throwing is reached, but the catch block in `sortNativeFunctionArgs` is not. So throwing the exception seems to kill the process. But I guess this is not a problem with scssphp since throwing an exception should generally assumed to be working.
Also, I encounter the problem on a Drupal installation, but I was unable to create a minimal example with only scssphp.
